django/core/paginator.py

```diff
@@ -52,6 +52,12 @@ def __init__(
             if error_messages is None
             else self.default_error_messages | error_messages
         )
+        if self.per_page <= self.orphans:
+            warnings.warn(
+                "The orphans value %d must be "
+                "smaller than the per_page value %d." % (self.orphans, self.per_page),
+                RuntimeWarning,
+            )
```

Comment:
This is a normal warning rather than a deprecation warning (note you will have to import `RemovedInDjango70Warning`):
```suggestion
            msg = (
                f"The orphans value {self.orphans} must be smaller than the "
                f"per_page value {self.per_page}."
            )
            # RemovedInDjango70Warning: When the deprecation ends, replace with:
            # raise ValueError(msg)
            warnings.warn(msg, RemovedInDjango70Warning, stacklevel=2)
```